eclipse-theia/theia
# branding: fix widget backgrounds (transparent)

#### What it does


Fixes: #8444 

The following pull-request updates the background of widgets in the framework so they are no longer `transparent`.
Previously, when there would be content behind views (such as a branding icon), the rendering would be incorrect.

The pull-request includes the following changes:
- fixes the background of views/widgets
- adds a branding example to `api-samples` (main menu icon and branding icon)

| Default (No Views Visible) | View Visible |
|:---:|:---:|
| <img width="1333" alt="Screen Shot 2020-08-31 at 9 29 26 AM" src="https://user-images.githubusercontent.com/40359487/91725584-f5241780-eb6c-11ea-9d0a-b527f59c2288.png"> | <img width="1333" alt="Screen Shot 2020-08-31 at 9 29 32 AM" src="https://user-images.githubusercontent.com/40359487/91725607-fbb28f00-eb6c-11ea-961b-f835ad412fe9.png"> |

#### How to test


1. start the example application.
2. verify that the menu icon is visible.
3. verify that the branding icon is visible (when no content is present in the main area).
4. verify (with multiple views/widgets) that the branding icon is not visible when content is present in the main area.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
